internal/pkg/describe/webapp.go

```diff
@@ -111,7 +111,7 @@ type ecsService interface {
 
 func (uri *WebAppURI) String() string {
 	if uri.Path != "" {
```

Comment:
Unless I am missing sth, could you add test for this logic as well?

Reply:
It looks like the test in webapp_test.go was changed: https://github.com/aws/amazon-ecs-cli-v2/blob/176852a160450ceb40983147f0a1f9a48f32fc15/internal/pkg/describe/webapp_test.go#L22

Were you looking for more @iamhopaul123 ?

Reply:
Oh that's the one I missed.